bode/bode/resources/tasks/schemas.py

```diff
@@ -11,7 +11,7 @@ class Meta:
     title = fields.String(validate=validate.Length(1, 80), required=True)
     description = fields.String(validate=validate.Length(0, 1024), default="")
     due_date = fields.DateTime(allow_none=True)
-    is_done = fields.Boolean(default=False)
+    status = fields.String(default="todo")
```

Comment:
This field should be validated:
`fields.String(validate=validate.OneOf(Status.list()), required=True)`